python/tank/deploy/tank_command.py

```diff
@@ -47,9 +47,9 @@
 ###############################################################################################
 # Built in actions (all in the tank_commands sub module)
 
-BUILT_IN_ACTIONS = [setup_project.SetupProjectAction, 
+BUILT_IN_ACTIONS = [setup_project.SetupProjectAction,
                     setup_project_wizard.SetupProjectFactoryAction,
-                    core_upgrade.CoreUpgradeAction, 
+                    core_upgrade.CoreUpdateAction,
```

Comment:
I assumed that the way the Desktop runs commands was the official one, which means that all these contants and classes are not visible from the public API.

Maybe I'm being a bit too pedantic about this, but this is supposed to be internal to core and the terminology wasn't accurate anymore, so I feel like it would be better to fix it.